assets/styles/variables/_section-titles.scss

```diff
@@ -203,15 +203,15 @@ $section-title-decoration-font-size: (epub:  0em, prince: 0em, web: 0em) !defaul
 /// @since 1.0.0
 $section-title-decoration-font-weight: normal !default;
 
-/// Defines the section title decoration margin bottom.
+/// Defines the section title decoration margin top.
 /// @type String | Map
 /// @since 1.0.0
-$section-title-decoration-margin-bottom: (epub:  0em, prince: 0em, web: 0em) !default;
+$section-title-decoration-margin-top:  (epub:  0em, prince: 0em, web: 0em) !default;
 
-/// Defines the section title decoration margin top.
+/// Defines the section title decoration margin bottom.
```

Comment:
Can you switch this with section title decoration margin top so there's no change?